eid_config.lua

```diff
@@ -175,7 +175,10 @@ EID.UserConfig = {
 	["ErrorColor"] = "ColorEIDError",
 	-- Enables or disables custom Descriptions for entities
 	-- Default = true
-	["EnableEntityDescriptions"] = true
+	["EnableEntityDescriptions"] = true,
+	-- Enables or disables item quality after item name
+	-- Default = true
+	["ShowQuality"] = true
```

Comment:
this new option also needs to be added to the default config at the bottom of the config file.

It also needs a new mod menu entry (just copy paste stuff around in the mod config menu file ;) )